src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+
+    <div class="conteiner">
+      <form class="search__form"
+            data-qa="big"
+            action="/"
+            methot="get">
```

Comment:
```suggestion
      <form 
        class="search__form"
        data-qa="big"
        action="/"
        methot="get"
      >
```

Reply:
fix indentation everywhere according to checklist
https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md